Fyne-cross fails on Mac to build web target due to os/user from dbus

<!-- Please search for open issues that relate to the same problem before opening a new one. -->

### Describe the bug:
fyne-cross fails on Mac to build web

```bash
% fyne-cross web -output myapp
[i] Target: web/
[i] Cleaning target directories...
[✓] "bin" dir cleaned: /app/fyne-cross/bin/web
[✓] "dist" dir cleaned: /app/fyne-cross/dist/web
[✓] "temp" dir cleaned: /app/fyne-cross/tmp/web
[i] Checking for go.mod: /Users/GolandProjects/fyne-poc/go.mod
[✓] go.mod found
[i] Packaging app...
# os/user
/go/pkg/mod/golang.org/toolchain@v0.0.1-go1.23.2.linux-arm64/src/os/user/lookup.go:22:41: undefined: current
/go/pkg/mod/golang.org/toolchain@v0.0.1-go1.23.2.linux-arm64/src/os/user/lookup.go:43:9: undefined: lookupUser
/go/pkg/mod/golang.org/toolchain@v0.0.1-go1.23.2.linux-arm64/src/os/user/lookup.go:52:9: undefined: lookupUserId
/go/pkg/mod/golang.org/toolchain@v0.0.1-go1.23.2.linux-arm64/src/os/user/lookup.go:58:9: undefined: lookupGroup
/go/pkg/mod/golang.org/toolchain@v0.0.1-go1.23.2.linux-arm64/src/os/user/lookup.go:64:9: undefined: lookupGroupId
/go/pkg/mod/golang.org/toolchain@v0.0.1-go1.23.2.linux-arm64/src/os/user/lookup.go:69:9: undefined: listGroups

error building application: exit status 1
[✗] could not package the Fyne app: could not package the Fyne app: exit status 1
```

### To Reproduce:
Steps to reproduce the behaviour:
1. Go to '...'
2. Run the command: fyne-cross web -output myapp
3. See error

### Example code:
Same code as here: https://github.com/fyne-io/fyne/blob/master/cmd/hello/main.go


### Device and debug info (please complete the following information):

<summary>Device info</summary>

 - **OS:** MacOs
 - **Version:** 15.1.1 (24B91)
 - **Go version:** 1.23.2
 - **fyne-cross version:** 1.5.0
 - **Fyne version:** 2.5.2 and v2.5.3-0.20241208134100-d1f7564b6632

<details><summary>Debug info</summary><br><pre>

runninb with debug flag, last lines before failure

```bash
fyne cli version: v2.3.5
/opt/homebrew/bin/docker run --rm -t -w /app -v /Users/GolandProjects/fyne-poc:/app -v /Users/user/Library/Caches/fyne-cross:/go --platform linux/arm64 --user 502 -e HOME=/tmp -v /run/host-services/ssh-auth.sock:/run/host-services/ssh-auth.sock -e SSH_AUTH_SOCK=/run/host-services/ssh-auth.sock -e CGO_ENABLED=1 -e GOCACHE=/go/go-build docker.io/fyneio/fyne-cross-images:web /usr/local/bin/fyne package -os web -name myapp -icon /app/fyne-cross/tmp/web/Icon.png -appBuild 1 -appVersion 1.0.0 -release
# os/user
/go/pkg/mod/golang.org/toolchain@v0.0.1-go1.23.2.linux-arm64/src/os/user/lookup.go:22:41: undefined: current
```


</pre></details>

--- a/internal/command/command.go
+++ b/internal/command/command.go
@@ -240,9 +240,6 @@ func fynePackageHost(ctx Context, image containerImage) (string, error) {
 	if ctx.CacheDirHost() != "" {
 		image.SetEnv("GOCACHE", ctx.CacheDirHost())
 	}
-	if image.OS() == webOS {
-		image.SetEnv("CGO_ENABLED", "0")
-	}
 
 	// run the command from the host
 	fyneCmd := execabs.Command(args[0], args[1:]...)
@@ -318,9 +315,6 @@ func fyneReleaseHost(ctx Context, image containerImage) (string, error) {
 
 	if ctx.CacheDirHost() != "" {
 		image.SetEnv("GOCACHE", ctx.CacheDirHost())
-	}
-	if image.OS() == webOS {
-		image.SetEnv("CGO_ENABLED", "0")
 	}
 
 	// run the command from the host

--- a/internal/command/web.go
+++ b/internal/command/web.go
@@ -69,6 +69,7 @@ func (cmd *web) Build(image containerImage) (string, error) {
 		return "", err
 	}
 
+	image.SetEnv("CGO_ENABLED", "0")
 	if cmd.defaultContext.Release {
 		// Release mode
 		err = fyneRelease(cmd.defaultContext, image)

